requirements.txt

```diff
@@ -1,3 +1,4 @@
 cdislogging==1.0.0
 Flask==1.0.2
 requests==2.20.0
+psqlgraph==3.0.1
```

Comment:
how come we need to add it? which dependency does it come from?

Reply:
it does not, and the crashing pidgin is probably caused by there is no penegrine, we have discussed this in a slack thread and i think this PR is likely going to be closed (as soon as the penegrine fix passes)